packages/hash/frontend/src/blocks/page/createSuggester/BlockSuggester.tsx

```diff
@@ -66,11 +68,56 @@ export const BlockSuggester: VFC<BlockSuggesterProps> = ({
           </div>
         </>
       )}
+      renderError={
+        blockFetchFailed ? (
+          <Box
+            sx={({ palette }) => ({
+              backgroundColor: palette.common.white,
+              position: "relative",
+            })}
+          >
+            <Box
+              sx={{
+                background:
+                  "linear-gradient(0deg, rgba(193, 207, 222, 0.36) -5%, rgba(255, 255, 255, 0) 50%)",
+                height: "40px",
+                width: "100%",
+                position: "absolute",
+                top: "-40px",
+                pointerEvents: "none",
+              }}
+            />
+            <Box
+              sx={{
+                display: "flex",
+                alignItems: "center",
+                p: 1,
+              }}
```

Comment:
I've defined it here instead to avoid applying the styles on the shadow defined here: https://github.com/hashintel/hash/pull/407#discussion_r835165710

Reply:
You don't need to do that. The shadow is absolutely positioned. You just need to add `left: 0` to the shadow and then you can apply these styles to the parent.

Reply:
Pretty simple to address this, but optional

Reply:
Addressed in this commit: https://github.com/hashintel/hash/pull/407/commits/a12662554d25ed615f487f89300006e64c1e5502. Imo, the specificity of styling was more clear in the previous version of this code, but we can do it this way for now.